.github/workflows/publish.yml

```
@@ -3,8 +3,7 @@ name: Publish Packages
 on:
   push:
     branches:
-      - "releases-slash/*"
-      - "releases-client/*"
+      - "releases/*"
```

Pour mieux respecter la convention :wink:
```suggestion
      - "release/*"
```
(comme pour le fichier de doc release, on peut le faire sur cette PR ou sur une PR dédiée, peu importe)

---

done